petcare-enterprise-apps/README.md

```diff
@@ -0,0 +1,25 @@
+# Pet Care Enterprise Applications
+
+## Pet Management Application (B2C)
```

Comment:
PetDesk Application for Pet Owners

Reply:
Addressed with 2dacf12da2870eaf5efbd6c93f79edf6f24d71ce.